### ISSUE TITLE
On the first load, the image is small in the product page image gallery.

### ISSUE THREAD

**Issue by marijak** [yireo/Yireo_Webp2#168]
Hi,

After upgrading to Magento version 2.4.6-p6, (php8.2)I we have this issue when a certain module is enabled:

1) Open any product page for the first time.
2) All gallery images appear in a small size.

ScreenRef: https://prnt.sc/Ovw9hbcQ9BTN

![image](https://github.com/yireo/Yireo_Webp2/assets/3399922/e95de75a-370e-4ab8-aa04-06859eb26fdb)

Do you have any suggestions on how to resolve this issue?

**Comment by jissereitsma** [comment]
No, unfortunately, if I try to replicate this issue, it does not occur in my own environment. Further details would be needed to let this be troubleshooted by other people.
